app/scripts/modules/core/src/presentation/Markdown.tsx

```diff
@@ -1,40 +1,41 @@
 import * as React from 'react';
 import { HtmlRenderer, Parser } from 'commonmark';
 import * as DOMPurify from 'dompurify';
+import './Markdown.less';
 
 export interface IMarkdownProps {
   [key: string]: any;
 
   /** markdown */
   message: string;
+
   /** optional tag */
   tag?: string;
+
+  /** The className(s) to apply to the tag (.Markdown class is always applied) */
+  className?: string;
 }
 
 /**
  * Renders markdown into a div (or some other tag)
  * Extra props are passed through to the rendered tag
  */
-export class Markdown extends React.Component<IMarkdownProps> {
-  public static defaultProps: Partial<IMarkdownProps> = {
-    tag: 'div',
-  };
-
-  private parser: Parser = new Parser();
-  private renderer: HtmlRenderer = new HtmlRenderer();
+export function Markdown(props: IMarkdownProps) {
+  const { message, tag: tagProp, className: classNameProp, ...rest } = props;
+  const tag = tagProp || 'div';
```

Comment:
Another option is to use a default prop for `tag` instead of aliasing then reassigning.